README.md

```diff
@@ -11,42 +15,60 @@ We have different feature files for unit and integration tests. The unit tests s
 
 # Test Descriptions
 
-| tag                             | description |
```

Comment:
Re-organizing the table:
* Separate into unit and integration tests
* Add missing tests
* Remove stale ones
* Alphabetize